internal/commands/internal/test/test.go

```diff
@@ -174,5 +254,8 @@ func NewTestCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVarP(&output, "output", "o", logs.DefaultFormat(), fmt.Sprintf("output format of logs, "+
 		"supported values: %s", strings.Join(logs.Formats(), ", ")))
 
+	cmd.PersistentFlags().StringVar(&reportFilename, "report", "",
+		"additionally write a report in Cirrus Annotation Format to this file")
```

Comment:
Should we add a link to the annotations repo? Or will it to weird to have a link in terminal output?

Reply:
I've added it and it looks OK, see https://github.com/cirruslabs/cirrus-cli/pull/417/commits/79ca5ecec14f05f30a1394830bf8b088a87539d0.

Plus we can always remove it in the future when the format becomes more widely known.